pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -1518,6 +1519,7 @@ fn failed_execution() {
             alice_did
         );
 
+        // Ensure affirms are in correct state.
```

Comment:
In the assertions below, two closures for user affirmations and for affirms received could be extracted (former also used above here), taking the status and DID.